js/battle.js

```diff
@@ -4027,12 +4035,14 @@ var Battle = (function () {
 				var item = Tools.getItem(args[2]);
 				var effect = Tools.getEffect(kwargs.from);
 				var ofpoke = this.getPokemon(kwargs.of);
-				poke.item = '';
+				poke.item = item.name;
```

Comment:
This is simple enough to fix, just have `.itemEffect` be a an array `[prevItem, effect]`.

`.itemEffect` is, incidentally, a horrible name for it. `.removedItem` is probably better.